File: JS/pyramid.js
const character = "#";
const count = 8;
const rows = [];

console.log("-".repeat(100))
console.log("THIS IS THE FIRST LOOP")

for (let i =  0; i < 10; i++) {
    // console.log(`With js:, ${i}`)
    rows.push(character.repeat(i))
    console.log(rows)
}


console.log("-".repeat(100))
console.log("THIS IS THE SECOND LOOP")

let result = ""

for ( const row of rows){
    console.log(row)
    console.log(typeof(row))
    result = result + row + "\n";
}

console.log("-".repeat(100))
console.log("THIS IS THE RESULT")

console.log(result)
console.log(typeof(result))

File: JS/pyramid.py
character = "#"
count = 8
rows = []

print("-" * 100)
print("THIS IS THE FIRST LOOP")

for i in range(10):
    # print(f"with python: {i}")
    rows.append(character)
    print(f'[{i}] {rows}')

print("-" * 100)
print("THIS IS THE SECOND LOOP")
result = ""

for i in rows:
    print(i)
    result = result + str(i) + "\n" 

print("-" * 100)
print("THIS IS THE RESULT")
print(result)
print(type(result))
